cl/corpus_importer/management/commands/scrape_pacer_free_opinions.py

```diff
@@ -204,7 +282,18 @@ def get_pdfs(options: OptionsType) -> None:
     q = cast(str, options["queue"])
     index = options["index"]
     cnt = CaseNameTweaker()
-    rows = PACERFreeDocumentRow.objects.filter(error_msg="").only("pk")
+    rows = PACERFreeDocumentRow.objects.filter(error_msg="")
```

Comment:
whats with the empty error msg here?  I see it's the same as before but its weird right?